www/js/template-list.js

```diff
@@ -17,10 +17,6 @@ module.exports = [
       "desc"  : "An example app for getting started with push notifications.",
       "npm"   : "phonegap-template-push@1.0.2"
     },
-    { "name"  : "React Hot Loader",
-      "desc"  : "Starter PhoneGap project using React.js, Babel, Webpack and Hot Reloading.",
-      "npm"   : "phonegap-template-react-hot-loader@1.3.0"
-    },
     { "name"  : "Star Track",
```

Comment:
We should also remove Star Track as this template no longer works due to API changes from Spotify.